### DISCUSSION TITLE
Use the same keymaps for opening splits in Neotree and Telescope

### DISCUSSION THREAD

**Discussion by dgmora** [LazyVim/LazyVim#558] [Ideas]
I have overwritten my neotree keymaps to 
```lua
      window = {
        mappings = {
          ["<C-v>"] = "open_vsplit",
          ["<C-x>"] = "open_split",
        },
      },
```
since this is how Telescope opens splits. Having the same mappings helps, and I find that `s` and `S` don't help me remember how's the split going to open. Do you think it'd make sense to have this as part of LazyVim directly? I think it should be easier for newer  people to remember. Although it's not the default behaviour for neotree and it also overwrites `C-x`, which is clear filter (I have never needed it).

**Comment by iamshm** [comment]
I am new to using lazy vim, where do I put this ?
should I create a file `neotree.lua` and in that
```
return {
 plugin-name, 
window = {
        mappings = {
          ["<C-v>"] = "open_vsplit",
          ["<C-x>"] = "open_split",
        },
      },
}
```
